.github/workflows/continuous-integration.yml

```diff
@@ -38,6 +38,10 @@ jobs:
         with:
           fetch-depth: 2
 
+      - name: "Temporarily remove support for PSR Log 3 on PHP 8.1"
+        run: 'sed -i "s/\"psr\/log\": \"^1|^2|^3\"/\"psr\/log\": \"^1|^2\"/" composer.json'
+        if: "${{ matrix.php-version=='8.1' }}"
```

Comment:
Why is this necessary? There is no occurrence of `psr/log` in `composer.json`.

Also, is it not something that could be more robust and legible with `composer require --no-update "psr/log ^1|^2"`?

Reply:
> Why is this necessary?

See the discussion starting at https://github.com/doctrine/dbal/pull/4967#discussion_r748911418.

> There is no occurrence of `psr/log` in `composer.json`.

We require `psr/log` as of this patch.

> Also, is it not something that could be more robust and legible with `composer require --no-update "psr/log ^1|^2"`?

As far as I can tell, `ramsey/composer-install` doesn't support `require`, and we don't install Composer as a binary.